website_sale_checkout_skip_payment/views/website_sale_template.xml

```diff
@@ -42,6 +42,13 @@
                 name="t-if"
             >not website_sale_order.amount_total and not website.checkout_skip_payment</attribute>
         </xpath>
+        <xpath expr="//div[@t-if='not (acquirers or tokens)']" position="attributes">
+            <attribute
+                name="t-if"
+                separator=" "
+                add=" and not website.checkout_skip_payment"
```

Comment:
Using the space in the `separator` attribute already, do you need to add a lead space in `add`?

Reply:
It's true so It's not needed this space in `add`. I fixed it.
Thank you! 😊